packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -591,6 +591,10 @@ class ComboBoxInternal extends React.Component<IComboBoxInternalProps, IComboBox
         : placeholderProp;
 
     const labelledBy = [this.props['aria-labelledby'], label && this._id + '-label'].join(' ').trim();
+    const labelProps = {
```

Comment:
This looks like the same code as before just pulled out into an object. How does this change help resolve the issue?

Reply:
makes it spreadable below

Reply:
yup, and also it's now used in two separate places, so I pulled it out.